Library/Homebrew/cask/lib/hbc/cli/style.rb

```diff
@@ -28,7 +28,8 @@ def install_rubocop
       begin
         Homebrew.install_gem_setup_path! "rubocop-cask", RUBOCOP_CASK_VERSION, "rubocop"
       rescue SystemExit
-        raise Hbc::CaskError, $stderr.string.chomp.sub("#{Tty.red}Error#{Tty.reset}: ", "")
+        require "utils/tty"
+        raise Hbc::CaskError, $stderr.string.chomp.extend(Tty::StringExtension).strip_ansi.gsub("Error: ", "")
```

Comment:
I think it's much cleaner to either make `strip_ansi` a module method or just monkeypatch `String`. It doesn't make sense to extend each individual string just to call this method.